Deep0Thinking/vscode-CP-codeforces
# v0.0.3

## [0.0.3]

### Changes

- Add more supported coding languages
- Add "RECENT" directory for problems
- Introduce icon for sorting in "DIFFICULTY", "RECENT" and "ALL" directories
- Introduce welcome page for login
- Store user-handle and sorting preferences in extension settings
- Implement user preferences for coding languages and template
- Include "contestId-index" in question filename
- Add tags in problem descriptions
- Default to keeping all folders collapsed
- Add notifications prompting users to check out extension settings
- Remove half-passed icon of problems for clarity
- Remove redundant files and code blocks
- Code formatting and cleanup

### Fixed

- Ensure solution files are created with absolute paths
- Check validity of user handle
